src/panels/beaconSummaryPanel/BeaconSummaryEditing.test.tsx

```diff
@@ -19,9 +19,6 @@ describe("BeaconSummaryEditing", () => {
     expect(
       await screen.findByDisplayValue(beaconFixture.model as string)
     ).toBeVisible();
-    expect(
-      await screen.findByDisplayValue(beaconFixture.type as string)
-    ).toBeVisible();
```

Comment:
`beaconFixture.type` is no longer editable, so remove the assertion that expects to find by the form input's current display value.